pkg/helper/machinepools/helpers.go

```diff
@@ -171,6 +172,52 @@ func taintValidator(val interface{}) error {
 	return fmt.Errorf("can only validate strings, got %v", val)
 }
 
+func GetAwsTags(
+	cmd *cobra.Command,
+	r *rosa.Runtime,
+	existingTags map[string]string,
+	inputTags []string,
+) map[string]string {
+	// Custom tags for AWS resources
+	tags := inputTags
+	delim := aws.GetTagsDelimiter(tags)
+	tagsList := map[string]string{}
+	if interactive.Enabled() {
+		if len(inputTags) == 0 {
+			for key, value := range existingTags {
+				tags = append(tags, fmt.Sprintf("%s%s%s", key, delim, value))
+			}
+		}
+		tagsInput, err := interactive.GetString(interactive.Input{
+			Question: "Tags",
+			Help:     cmd.Flags().Lookup("tags").Usage,
+			Default:  strings.Join(tags, ","),
+			Validators: []interactive.Validator{
+				aws.UserTagValidator,
+				aws.UserTagDuplicateValidator,
+			},
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid set of tags: %s", err)
+			os.Exit(1)
+		}
+		if len(tagsInput) > 0 {
+			tags = strings.Split(tagsInput, ",")
+		}
+	}
+	if len(tags) > 0 {
+		if err := aws.UserTagValidator(tags); err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
```

Comment:
@gdbranco I think we should start returning errors from functions and not `os.Exit()` as it just makes them impossible to test.